__tests__/app.test.js

```diff
@@ -81,4 +81,37 @@ describe('Error Handling 400/404', () => {
       });
   });
 });
-
+describe("GET /api/articles", () => {
+  test("return 200 status, will get all the articles in descending order with a comment count and no body property", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        console.log(articles);
+        expect(Array.isArray(articles)).toBe(true);
+        articles.forEach((article) => {
```

Comment:
the assertions in the callback won't run if the array length is zero. Can you ensure this is the case?

Reply:
All done, should be all good now